python/lsst/afw/geom/transform/transformContinued.py

```diff
@@ -33,10 +35,20 @@ def getJacobian(self, x):
                     self.getFromEndpoint().getNAxes())
     return matrix
 
+
+def of(self, before):
+    if isinstance(before.getToEndpoint(), type(self.getFromEndpoint())):
```

Comment:
Using the equality test didn't occur to me, that is cleaner.

The original code had `type(...) == type(...)`, but `flake8` didn't like it.